src/api/lifespan.py

```diff
@@ -6,28 +6,39 @@
 
 from src.config import api_settings
 from src.storage.sql import SQLAlchemyStorage
+import asyncio
```

Comment:
Here is incorrect sorting of imports.

https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

In JetBrains you can do Ctrl + Alt + O, dunno how in VScode.


May be we can add isort to pre-commit config, but I bump into error "No module named 'pkg_resources'"

Reply:
fixed for now, I will try to add isort pre-commit after this feature

Reply:
![image](https://github.com/user-attachments/assets/83aa36da-a78a-4207-b7c5-3e7fd64e0d83)
why not this just to use this in ruff configuration in pyproject.toml? It will sort all imports